releasenotes/notes/logs-agent-payload-80ed60e3ad556ec9.yaml

```diff
@@ -0,0 +1,4 @@
+---
+features:
+  - |
+    Change the payload sent by the agent from raw text formatted using the syslog protocol to a protocol buffer payload.
```

Comment:
I am not sure we need a release note here since this change does not impact the user experience, you can just add the noreno label to your PR.